ciscoise.json

```diff
@@ -1,33 +1,35 @@
 {
     "appid": "85232924-4776-4896-88fa-b982b304af7c",
     "name": "Cisco ISE",
-    "description": "This app implements investigative and containment actions on a Cisco ISE device.",
-    "publisher": "Phantom",
+    "description": "This app implements investigative and containment actions on a Cisco ISE device",
+    "publisher": "Splunk",
     "type": "network access control",
-    "main_module": "ciscoise_connector.pyc",
-    "app_version": "1.2.20",
+    "main_module": "ciscoise_connector.py",
+    "app_version": "2.0.0",
     "utctime_updated": "2018-12-07T02:21:36.000000Z",
     "package_name": "phantom_ciscoise",
     "product_vendor": "Cisco Systems",
     "product_name": "Cisco ISE",
-    "product_version_regex": "2.2",
-    "min_phantom_version": "3.0.190",
+    "product_version_regex": "/([2].[67])|([3].[01])/",
+    "min_phantom_version": "5.0.0",
     "logo": "logo_cisco.svg",
     "logo_dark": "logo_cisco_dark.svg",
-    "license": "Copyright (c) Phantom Cyber Corporation, 2014-2018",
+    "license": "Copyright (c) 2016-2021 Splunk Inc.",
```

Comment:
Copyright years are mismatching. It should be either `2016-2021` or `2014-2021` across all the files.

Reply:
Updated

Reply:
@pdesai-crest, It looks like release_notes has the first entry for 2016 and we have updated with 2014-2021 in all the files. Can you check with the US team?